modules/Bio/EnsEMBL/Compara/PipeConfig/EPOwith2x_conf.pm

```diff
@@ -0,0 +1,302 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+=head1 NAME
+
+Bio::EnsEMBL::Compara::PipeConfig::EPOwith2x_conf
+
+=head1 SYNOPSIS
+
+    init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::EPOwith2x_conf.pm -host mysql-ens-compara-prod-X -port XXXX \
+        -division $COMPARA_DIV -species_set_name <species_set_name> -low_epo_mlss_id <id> -high_epo_mlss_id <id>
+
+=head1 DESCRIPTION
+
+    This pipeline runs EPO and EPO2x together. For more information on each pipeline, see their respective PipeConfig files:
+    - Bio::EnsEMBL::Compara::PipeConfig::EPO_conf
+    - Bio::EnsEMBL::Compara::PipeConfig::EpoLowCoverage_conf
+
+=head1 CONTACT
+
+Please email comments or questions to the public Ensembl
+developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
+
+Questions may also be sent to the Ensembl help desk at
+<http://www.ensembl.org/Help/Contact>.
+
+=cut
+
+package Bio::EnsEMBL::Compara::PipeConfig::EPOwith2x_conf;
+
+use strict;
+use warnings;
+
+use Bio::EnsEMBL::Hive::Version 2.4;
+use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf;   # For INPUT_PLUS
+
+use Bio::EnsEMBL::Compara::PipeConfig::Parts::EPOMapAnchors;
+use Bio::EnsEMBL::Compara::PipeConfig::Parts::EPOAlignment;
+use Bio::EnsEMBL::Compara::PipeConfig::Parts::EpoLowCoverage;
+
+use base ('Bio::EnsEMBL::Compara::PipeConfig::ComparaGeneric_conf');
+
+sub default_options {
+    my ($self) = @_;
+
+    return {
+        %{$self->SUPER::default_options},
+
+        'pipeline_name' => $self->o('species_set_name').'_epo_'.$self->o('rel_with_suffix'),
+        'division'      => 'vertebrates',
```

Comment:
Division is passed by the initialisation command line (`division = 'vertebrates'` should be in `Vertebrates::EPOwith2x_conf` pipeconfig file).

Reply:
I think this is a sensible default (since we don't run this pipeline on anything other than verts right now). There is not division specific config for the same reason

Reply:
I agree with @JAlvarezJarreta . The convention that's been followed in his pull-requests (and that will be written down in Confluence) is that pipelines that are directly under PipeConfig shouldn't have `division` set, it needs to be passed on the command-line. Only pipelines that are in a sub-directories will have the `division` set accordingly.